### ISSUE TITLE
CitrusPay integration

### ISSUE THREAD

**Issue by sonali19** [citruspay/open-ios#1]
I have integrated citruspay in my IOS application. I have changed the keys and BillURL but while doing a transaction I am getting the response as "Invalid signature key" . Can anyone help me out of this.

Thanks in advance .

**Comment by YadneshCitrus** [comment]
Hi Sonali
which merchant are you working for?

**Comment by sonali19** [comment]
I am trying for payment gateway integration in my app with "citruspay"

**Comment by YadneshCitrus** [comment]
Firstly you need to register with Citrus as a merchant, after that Citrus will issue keys for you
Please mail Citrus  MobileDev@CitrusPay.Com
Also go through http://citruspay.com/DevelopersGuide/#/ for more information

**Comment by sonali19** [comment]
Yes I have registered and obtained the keys which I provided in the test application. After getting the bill details while I try to do a transaction with the test application provided I am getting the response as " invalid signature key" with code 111

**Comment by YadneshCitrus** [comment]
mail me all the merchant  details. name, sandbox keys 

 to: yadnesh.wankhede@citruspay.com
 CC: mobiledev@citruspay.com

we dont support here on github

**Comment by somsin** [comment]
Hi  am facing this issue to use in ios . help me solve this. 
billJson <?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE HTML SYSTEM "about:legacy-compat">
<html><div version="2.0"></div><body><div id="page-wrapper"><div id="page-client-logo"> </div><div class="h-space">  </div><div version="2.0" id="footer"><link href="/resources/pg/css/error/error.css" type="text/css" rel="stylesheet"/><div id="page-content"><div class="box-white fullerror"><h2>Access is denied</h2><div class="hrsep"> </div><div class="transDetails">
				Bad Request:Following parameters are missing-Order Amount,Transaction ID,Return URL
			</div><div class="clear"/></div></div></div><div class="clear"> </div></div></body></html>
 [CitrusPay trace] signature (null) 
 [CitrusPay trace] validation error 0
